src/components/List/List.stories.tsx

```diff
@@ -32,19 +32,35 @@ const itemGroups = [
   "Villagers",
 ];
 
-const ListTemplate = ({ listType, items, ...args }) => (
-  <List type={listType} {...args}>
-    {items.map((item, i) => (
-      <li key={i}>{...item}</li>
-    ))}
+const definitions = [
```

Comment:
One of the most common use cases for DL is a list of links - it would be super helpful to have that as an example 
![Screen Shot 2021-04-02 at 2 47 37 PM](https://user-images.githubusercontent.com/3579472/113444774-8937b500-93c2-11eb-8792-cd6242b2d19a.png)

Reply:
Sure, no problem. I will add.